ports/vulkan-loader/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "vulkan-loader",
+  "version": "1.3.221",
+  "description": "Provides the Khronos official Vulkan ICD desktop loader for Windows, Linux, and MacOS.",
+  "homepage": "https://github.com/KhronosGroup/Vulkan-Loader",
+  "license": "Apache-2.0",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    "vulkan-headers"
```

Should `vulkan-headers` match this port's version? 

```suggestion
    {
        "name": "vulkan-headers",
        "version>=": "1.3.221"
    }
```